elements/pfe-progress-indicator/demo/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>PatternFly Element | pfe-progress-indicator Demo</title>
+
+  <noscript>
+    <link href="../../pfelement/pfelement-noscript.min.css" rel="stylesheet">
+  </noscript>
+
+  <link href="../../pfelement/pfelement.min.css" rel="stylesheet">
+
+  <!-- uncomment the es5-adapter if you're using the umd version -->
+  <script src="/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+  <script src="/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.min.js"></script>
+  <script>require(['../pfe-progress-indicator.umd.js'])</script>
+</head>
+
+<body unresolved>
+  <h1>&lt;pfe-progress-indicator&gt;</h1>
+  <pfe-progress-indicator pfe-indeterminate>
```

Comment:
Instead of requiring the light-dom to provide the initial state, can the web component set it?

Reply:
What about using aria attributes instead of a custom attribute? https://stackoverflow.com/questions/31716425/how-do-you-make-the-loading-icon-accessible-for-screen-readers-like-jaws

Reply:
I actually looked into aria-busy, but when I looked at what it does it didn't seem like the right choice. The official write up about it says something "Users of assistive technology will not be able to read anything while the content is getting updated." This is a message that a screen reader could actually see and read.

Reply:
@castastrophe Let me know if we want to toss this idea around some more. For now I'm marking resolved.